crates/ide/src/static_index.rs

```diff
@@ -230,7 +230,11 @@ impl StaticIndex<'_> {
         self.files.push(result);
     }
 
-    pub fn compute<'a>(analysis: &'a Analysis, workspace_root: &VfsPath) -> StaticIndex<'a> {
+    pub fn compute<'a>(
+        analysis: &'a Analysis,
+        workspace_root: &VfsPath,
+        exclude_vendored_libraries: bool,
```

Comment:
up to you, but rust-analyzer has a lot of booleans that might be better served as enums. what do you think about making this something like:

```rust
enum VendoredLibraries {
    Include,
    Exclude
}
```

Reply:
if `workspace_root` is only used when vendored libraries are included, maybe something like this would work:

```rust
enum VendoredLibraries<'a> {
    Included { workpace_root: &'a VfsPath },
    Excluded,
}
```